moto-hses-client/README.md

```
@@ -51,6 +51,7 @@ The following robot models have been tested and verified for compatibility:
 | 0x300 | Plural I/O Data Reading / Writing Command |
 | 0x301 | Plural Register Data Reading / Writing Command |
 | 0x302 | Plural Byte Type Variable (B) Reading / Writing Command |
+|| 0x303 | Plural Integer Type Variable (I) Reading / Writing Command |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)